api/app/src/main/kotlin/packit/repository/PacketRepository.kt

```diff
@@ -10,4 +10,7 @@ interface PacketRepository : JpaRepository<Packet, String>
 {
     @Query("select p.id from Packet p order by p.id asc")
     fun findAllIds(): List<String>
+
+    @Query("select p from Packet p order by p.time desc limit 1")
```

Comment:
Using the JPA built in method syntax (like `findTopOrderByTimeDesc`) gives a `SqlGrammarException` and I can't for the life of me work out why. The Postgres dialect seems to be set correctly and the error is really opaque. So I gave up and did this instead.